packages/render-fidelity-tools/src/common.ts

```diff
@@ -253,6 +279,13 @@ export class ImageComparator {
       }
     }
 
+    const imagePixelCount = width * height;
+    if (whitePixelCount === imagePixelCount ||
+        blackPixelCount === imagePixelCount ||
+        transparentPixelCount === imagePixelCount) {
```

Comment:
If an image is rendered correctly but with 50% transparency, then `transparentPixelCount` will still equal `imagePixelCount` because you've defined "transparent pixels" as anything < 100% opaque. Is this what you want? In that case the error "Candidate image is colorless" might be misleading"

Reply:
My recommendation for this logic would be to pre-multiply the color channels by alpha, and _then_ test for white or black pixels. This way, both [0, 0, 0, 255] and [11, 22, 33, 0] register as "black"

Reply:
That's a good idea, and simple.

Reply:
Thank you for the recommendation and it's really much better than mine.  We found an error that model-viewer's golden is captured when the poster hasn't faded away and that's why i thought checking all pixels' alpha this way would be helpful.  
![image](https://user-images.githubusercontent.com/44676379/90086503-90516c00-dce8-11ea-9d3c-96a977cc3235.png)
But yeah the error message is misleading in this case. Besides, this bug is probably fixed right now, so i removed this check in latest commit.  @elalish Do you still want this check to make sure the poster has faded away when we take screenshots?